src/jit/regset.h

```diff
@@ -99,7 +99,7 @@ class RegSet
     void verifyRegistersUsed(regMaskTP regMask);
 
 public: // TODO-Cleanup: Should be private, but GCInfo uses them
-    __declspec(property(get = GetMaskVars, put = SetMaskVars)) regMaskTP rsMaskVars; // mask of registers currently
+
                                                                                      // allocated to variables
```

Comment:
`// allocated to variables` that a leftover from the previous line, please delete it as well.